backend/zeno_backend/database/insert.py

```diff
@@ -141,10 +149,10 @@ def label(label_spec: LabelSpec, project: str):
                 )
             )
         db.execute(
-            sql.SQL("UPDATE {} SET label = %s WHERE item = %s;").format(
+            sql.SQL("UPDATE {} SET label = %s WHERE data_id = %s;").format(
                 sql.Identifier(project)
             ),
-            [label_spec.label, label_spec.item],
+            [label_spec.label, label_spec.data_id],
```

Comment:
Here you call it data_id. This probably won't work because of the different names in the DB.